va/va_dec_jpeg.h

```diff
@@ -70,6 +70,10 @@ typedef struct _VAPictureParameterBufferJPEGBaseline {
     }                   components[255];
     /** \brief Number of components in frame (Nf). */
     unsigned char       num_components;
+    /** \brief Input color space 0: YUV, 1: RGB, 2: BGR, others: reserved */
+    unsigned char       color_space;
+    /** \brief Set to VA_ROTATION_* for a single rotation angle reported by VAConfigAttribDecJPEG. */
+    unsigned int 	rotation;
 } VAPictureParameterBufferJPEGBaseline;
```

Comment:
How about to define VAConfigAttribDecJPEG as read/write attribute? User can use VAConfigAttribDecJPEG to query the valid color space and rotation, then set the required value via VAConfigAttribDecJPEG when decoding JPEG image. If so, we won't  change VAPictureParameterBufferJPEGBaseline.